test/e2e/framework/scaletest/templates/networkpolicy.go

```diff
@@ -0,0 +1,24 @@
+package templates
+
+import (
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	NetworkPolicy = netv1.NetworkPolicy{
```

Comment:
+1 this doesn't necessarily seem scale test specific, let's put it in a more common directory and/or reuse with existing create deny netpol steps

Reply:
I can put it in the folder `test/e2e/framework/kubernetes/templates` and existing create deny netpol step can be changed in another task to reuse it and perform changes needed. Is that ok?